src/libs/service-alerts-polling.js

```diff
@@ -0,0 +1,15 @@
+import { getServiceAlerts, serviceAlertsStore } from 'src/libs/service-alerts'
+
+
+export const startPollingServiceAlerts = () => {
+  getServiceAlerts().then(serviceAlerts => serviceAlertsStore.set(serviceAlerts))
+
+  const interval = setInterval(() => {
+    try {
+      getServiceAlerts().then(serviceAlerts => serviceAlertsStore.set(serviceAlerts))
+    } catch (error) {
```

Comment:
Tried to use `withErrorIgnoring` here, but ran into a circular import issue with error.js and auth.js.

Reply:
I like the concision and clarity of `withErrorIgnoring`. Could be worth a follow-up to break the cycle.